firmware/quintuna/CRIT/src/io/io_rotary.c

```diff
@@ -0,0 +1,89 @@
+#include "hw_gpio.h"
+#include "hw_gpios.h"
```

Comment:
idk what's the best way to handle this, but I think there should be an onPressed, onRight, onLeft type system which can then drive the display

Reply:
or perhaps some mechanism by which edges are notified

Reply:
rather than making decisions based on the current state